move_action_node:
  ros__parameters:
    main: [0, 0]
    corr1: [3.25, 16.5]
    corr2: [-5.5, 16.5]
    corr3: [4 , -12.5]
    corr4: [-6 , -12.5]
    s1: [6.25 , 13]
    s2: [-8.75 , 13]
    z1: [9, 9.77]
    z2: [9, 4.09]
    z3: [-10.9, 9.19]
    z4: [-11.4, 4.05]
    o1: [0.34, -4.48]
    o2: [-2.51, -4.61]
    o3: [0.58, -18.2]
    o4: [-3.73, -17.8]
    g1: [6.89, -6.87]
    g2: [7.19, -21.2]
    g3: [-0.92, -28.9]
    g4: [-9.21, -6.54]
    g5: [-9.15, -21]
    b1: [0.41 , -12.4]
    b2: [-3 , -12.3]
    w1: [-10.1 , -27.5]
    